pulp_ansible/tests/functional/utils.py

```diff
@@ -63,7 +63,7 @@ def gen_ansible_remote(url=ANSIBLE_FIXTURE_URL, **kwargs):
     :param url: The URL of an external content source.
     """
     if "rate_limit" not in kwargs:
-        kwargs["rate_limit"] = 10
+        kwargs["rate_limit"] = 5
```

Comment:
Did we have problems with it at 10? I'm asking because if we can make the tests go twice as fast that would be good.

Reply:
that was my first attempt before I realize the rate_limit was not being set,
I'll keep it 10